endpoints.json

```diff
@@ -75,5 +75,24 @@
                 }
             ]
         }
+    },
+
+    "POST /api/articles/:article_id/comments":
+    {
+        "description": "Sends a comment to the database.",
+        "queries": [],
+        "format": { "username": "<string of an existing user>", "body": "<any string>" },
```

Comment:
I would recommend renaming this key from "format" to something more along the lines of requested body